pkg/debugapi/balances.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package debugapi
+
+import (
+	"github.com/ethersphere/bee/pkg/jsonhttp"
+	"github.com/ethersphere/bee/pkg/swarm"
+	"github.com/gorilla/mux"
+
+	"net/http"
+)
+
+type balanceResponse struct {
+	Peer    string `json:"peer"`
+	Balance int64  `json:"balance"`
+}
+
+type balancesResponse struct {
+	Balances []balanceResponse `json:"balances"`
+}
+
+func (s *server) balancesHandler(w http.ResponseWriter, r *http.Request) {
+
+	balances, err := s.Accounting.Balances()
+
+	if err != nil {
+		jsonhttp.InternalServerError(w, "Can not get balances")
+		s.Logger.Debugf("debug api: balances: %v", err)
+		s.Logger.Errorf("debug api: balances: %v", err)
+		return
+	}
+
+	var balResponses []balanceResponse
+
+	for k := range balances {
+		balResponses = append(balResponses, balanceResponse{
+			Peer:    k,
+			Balance: balances[k],
+		})
+	}
+
+	jsonhttp.OK(w, balancesResponse{Balances: balResponses})
+
+}
+
+func (s *server) peerBalanceHandler(w http.ResponseWriter, r *http.Request) {
+	// TODO: Currently, we do not check the length of the hex address (such as prepending zeroes), should we?
```

Comment:
this `TODO` can be removed. if such check is made, it can be made inside `ParseHexAddress`, returning an error to the calling context if need be